src/aggregator/aggregator/entry.go

```diff
@@ -1116,22 +1112,21 @@ func (e *Entry) checkLatenessForForwardedMetric(
 
 	e.metrics.forwarded.arrivedTooLate.Inc(1)
 
-	return nil
-	//if !e.opts.VerboseErrors() {
-	//	// Don't return verbose errors if not enabled.
-	//	return errArrivedTooLate
-	//}
-	//
-	//timestamp := time.Unix(0, metricTimeNanos)
-	//pastLimit := time.Unix(0, currNanos-maxLatenessAllowed.Nanoseconds())
-	//err := fmt.Errorf("datapoint for aggregation forwarded too late: "+
-	//	"id=%s, off_by=%s, timestamp=%s, past_limit=%s, "+
-	//	"timestamp_unix_nanos=%d, past_limit_unix_nanos=%d",
-	//	metric.ID, maxLatenessAllowed.String(),
-	//	timestamp.Format(errTimestampFormat),
-	//	pastLimit.Format(errTimestampFormat),
-	//	timestamp.UnixNano(), pastLimit.UnixNano())
-	//return xerrors.NewRenamedError(errArrivedTooLate, err)
+	if !e.opts.VerboseErrors() {
+		// Don't return verbose errors if not enabled.
+		return errArrivedTooLate
+	}
+
+	timestamp := time.Unix(0, metricTimeNanos)
+	pastLimit := time.Unix(0, currNanos-maxLatenessAllowed.Nanoseconds())
+	err := fmt.Errorf("datapoint for aggregation forwarded too late: "+
```

Comment:
Does `forward` mean the data point is from another (maybe the same) aggregator instance? Can you add a piece of comment to explain it?

Reply:
Yep, it is from other aggregators/same aggregator for `aggregation step`.